src/index.html

```diff
@@ -13,5 +13,84 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post" onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal Information</legend>
+          <div>
+            <input type="text" name="sname" id="sname" autocomplete="off" required>
```

Comment:
more than 2-3 attributes, better to move to the new line

```suggestion
            <input
              type="text"
              name="sname"
              id="sname"
              autocomplete="off"
              required
            >
```

Reply:
fix all inputs